manuscript/manuscript.md

```diff
@@ -70,14 +76,38 @@ Since analytical and data processing procedures may improve or otherwise change
 
 # Rule 4: Rule {Store data in open formats}
 
-To maximize accessibility and long-term value, data should be stored in file formats whose specificiations are freely-available. The exact file type will depend on the type of data being stored (e.g. numeric measurements, text, images, video) but the key idea is that data should not require proprietary software or hardware to access. Proprietary formats can change, maintaining organizations can go out of business, and changes in license fees could make access to data in  proprietary formats simply unaffordable. Examples of open data formats include 
+To maximize accessibility and long-term value, data should be stored in file formats whose specificiations are freely-available. The exact file type will depend on the type of data being stored (e.g. numeric measurements, text, images, video) but the key idea is that data should not require proprietary software or hardware to access. Proprietary formats can change, maintaining organizations can go out of business, and changes in license fees could make access to data in  proprietary formats simply unaffordable. Examples of open data formats include
 * CSV for tabular data
 * HDF5 for (??)
 * ?? for images
 * (help me out here folks, it's been a long week)
 and examples of closed formats include XLSX, DICOM, (again need more examples). At a minimum, data being stored for archival purposes should be stored in open formats, even if day-to-day processing uses closed formats.
 
-# Rule 5: Rule {-}
+# Rule 5: Rule {Data should be uniquely identifiable}
+
+To aid reproducibility, the data used in a scientific publication should be uniquely identifiable.
+Ideally, datasets should have a unique identifier such as a Document Object Identifier (DOI), Archival Resource Key (ARK), or a Persistant URL (PURL).
+An increasing number of online services, such as [Figshare](http://figshare.com/) are able to provide these.
+
+Datasets may evolve over time and in order to distinguish between different versions of the same data *Semantic versioning* should be used.
+An example of this can be seen in the CellPack datasets [@johnson2014cellpack].
+
+A semantic version number takes the form: `Major.Minor.Patch`, e.g. `0.2.7`.
```

Comment:
Thanks for your feedback. I've pushed some changes to this PR which (hopefully) answer most of what you have written here. I'm not entirely sure I understood the change you are suggesting in this note though, perhaps you were intending to remove more text than I have?